src/react-lab.tsx

```diff
@@ -21,9 +29,6 @@ interface IProps {
   onModelLoad?: () => void;
   onPropChange?: (propName: string, value: any) => void;
   onLogEvent?: (action: string, data: any) => void;
-  // Source to Lab embeddable page. Needs to be under the same domain as the application.
-  // This package is providing lab distribution that can be used (/lab).
-  embeddableSrc?: string;
```

Comment:
Should we add a @deprecated JSDoc tag to this property? Something like 
```
/**
 * @deprecated since version 1.0.0 - use labDistPath property with a path to the lab folder instead
 */
```

I've never had to mark anything as deprecated before but from brief research it looks like it could maybe help devs upgrading other projects

Reply:
It'd be nice, but I think it's a bit different case. This property isn't just deprecated. It's not supported anymore (and it can't really be). That's why I bumped up the major version.

So, leaving it in IProps interface is confusing. We'll have an unsupported prop, and the code will compile fine in somebody's project, even though it will be broken in fact.

Personally, I'd prefer if my complication just breaks and I can get a clear TypeScript compilation error about the unsupported prop. Otherwise, the developer would need to dig into this comment somehow, which doesn't seem likely if he doesn't know that he should look for it.

Hopefully, we probably don't have too many projects using 0.x version anyway (thermoscope and that's all?).